spec/map-prototype-reduce.html

```diff
@@ -0,0 +1,22 @@
+<p>When the `reduce` method is called, the following steps are taken:</p>
+
+<emu-alg>
+    1. Let _map_ be the *this* value.
+        1. If Type(_map_) is not Object, throw a *TypeError* exception.
+        2. If _map_ does not have a [[MapData]] internal slot, throw a *TypeError* exception.
+    2. If IsCallable(_callbackfn_) is false, throw a *TypeError* exception.
+    3. Let _entries_ be the List that is _map_.[[MapData]].
```

Comment:
Should we iterate over `[[MapData]]` or use `@@iterator`? 🤔

Reply:
`[[MapData]]` makes sense for consistency with another methods.